src/main/java/core/basesyntax/BallNumberSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class BallNumberSupplier {
+    public int getRandomNumber() {
+        return new Random().nextInt(100);

```

Comment:
create a global field Random and use it here, so you won't need to create its instance each time this method is called